src/routes/owners/owners.controller.ts

```diff
@@ -0,0 +1,23 @@
+import { Controller, Get, Param } from '@nestjs/common';
+import { ApiOkResponse, ApiTags } from '@nestjs/swagger';
+import { SafeList } from '../../domain/safe/entities/safe-list.entity';
```

Comment:
We don't need the domain entity at this level.

The `service` is already returning the route entity too.

Reply:
Right, thanks! 

Fixed in [f5d0b9f](https://github.com/5afe/safe-client-gateway-nest/pull/192/commits/f5d0b9f1c62a66c36908774dc76d43563d2f0631)